### DISCUSSION TITLE
I don't understand the putput of align_optimal

### DISCUSSION THREAD

**Discussion by etiur** [biotite-dev/biotite#549] [Q&A] [answered]
Thanks in advance

So I provided 2 proteins sequences

Sequence 1
RACDLMANVEVRLVRPLSNGIWKQELLKLRDKIQWAFPWQQYEAVEYGCYAQNLWRSGSKMQNAVYLTMWEKYMTGCRRHVCAKQITRCKPGMFYRGCCTANTQTHHDVWWAQPTPFDYGGKDCWCKTIRVYNPGNWPHGRSKHAWGFEAAFFKGFNKLDHPPACLCWDSGLQWGQPLGYICYCFKISRWASSEMPHHKGRNGCSKIHFDFYPSAGAFTWNMDLYVAVWCSFEPQSPKRREIPSAPFYFTGHMDTWFACEYHYVTKCDRSLDKITCGLKIHFLQCMNGVDKSFAFQAHPV

sequence 2
SKSFMDIMYGAMSGSITANMMHRMWGCRPSFVSQIKFHSPDVLIGHDVVKESYTKKVMKFSWAPIDGPECFWWERCNNVPGRTHMKGPSPWQSANLTGFINSYHQPHRFHDSFWEFDCWRKNEGLIVPNDVTKCKKHYCNELGNQCYFSAPCNIIDMKWMHGWWMVEPPGKVQLELTDTNCGLTDPYHQFSRQSVPRIDSTHKYDTVRFRLNIKKQIQRQEWGGNWVWTEIRCVIWCYVNAYYQNDWLFLCQQGAYGIWSHGVIFYMPSNPWQQTFKYELESCYTIAANGQDCSSEFVVV

And this fucntion to generate a pairwise alignment

def pairwise(sequ1, sequ2):
    seq1 = seq.ProteinSequence(sequ1)
    seq2 = seq.ProteinSequence(sequ2)
    matrix = align.SubstitutionMatrix.std_protein_matrix()
    alignments = align.align_optimal(seq1, seq2, matrix, local=False)
    return alignments

The it returns a list of 3 elements, what are each of the elements?

If I want to use the alginments later to calculate the pari wise sequence identity which element whould I choose?

**Comment by padix-key** [comment, marked as answer]
Sometimes (like in your case) there a multiple optimal alignments possible, i.e. multiple alignments with the same maximum reachable score. Hence, all of the alignments in the returned list are 'correct'. Hence, the decision which alignment to proceed with is arbitrary. I would assume that the sequence identity will be very similar across these alignments.
